src/read_write/ply.rs

```diff
@@ -403,9 +403,11 @@ impl PlyIterator {
         }
 
         // We align the buffer of this 'BufReader' to points, so that we can index this buffer and know
-        // that it will always contain full points to parse.
+        // that it will always contain full points to parse. Capacity should be approximately 1MB
+        let num_points_in_mb: usize = BUFFER_CAPACITY / num_bytes_per_point;
```

Comment:
This should probably not use `BUFFER_CAPACITY`, as it may be different to one MB.